Index: meeting-server/src/main/java/com/fan/server/service/impl/EnterpriseServiceImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.fan.server.service.impl;\r\n\r\nimport com.fan.server.pojo.Enterprise;\r\nimport com.fan.server.mapper.EnterpriseMapper;\r\nimport com.fan.server.service.IEnterpriseService;\r\nimport com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;\r\nimport org.springframework.stereotype.Service;\r\n\r\n/**\r\n * <p>\r\n *  服务实现类\r\n * </p>\r\n *\r\n * @author BestOfTheWorld\r\n * @since 2023-07-05\r\n */\r\n@Service\r\npublic class EnterpriseServiceImpl extends ServiceImpl<EnterpriseMapper, Enterprise> implements IEnterpriseService {\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/meeting-server/src/main/java/com/fan/server/service/impl/EnterpriseServiceImpl.java b/meeting-server/src/main/java/com/fan/server/service/impl/EnterpriseServiceImpl.java
--- a/meeting-server/src/main/java/com/fan/server/service/impl/EnterpriseServiceImpl.java	(revision de4ecccb6acab0c87522004db7484db0c4aa98f8)
+++ b/meeting-server/src/main/java/com/fan/server/service/impl/EnterpriseServiceImpl.java	(date 1688554565246)
@@ -4,6 +4,7 @@
 import com.fan.server.mapper.EnterpriseMapper;
 import com.fan.server.service.IEnterpriseService;
 import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;
+import org.apache.ibatis.annotations.Mapper;
 import org.springframework.stereotype.Service;
 
 /**
Index: meeting-server/src/main/resources/application.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>server:\r\n  port: 9090\r\n\r\nspring:\r\n  datasource:\r\n    driver-class-name: com.mysql.cj.jdbc.Driver\r\n    username: root\r\n    password: 666666\r\n    url: jdbc:mysql:///internship_platform\r\n  redis:\r\n    port: 6379\r\n    host: localhost\r\n  mvc:\r\n    pathmatch:\r\n      matching-strategy: ant_path_matcher\r\n\r\nlogging:\r\n  level:\r\n    com.fan: debug\r\n\r\nmybatis-plus:\r\n  type-aliases-package: com.fan.*.pojo\r\n  mapper-locations: classpath:/mapper/*Mapper.xml
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/meeting-server/src/main/resources/application.yml b/meeting-server/src/main/resources/application.yml
--- a/meeting-server/src/main/resources/application.yml	(revision de4ecccb6acab0c87522004db7484db0c4aa98f8)
+++ b/meeting-server/src/main/resources/application.yml	(date 1688554504757)
@@ -5,7 +5,7 @@
   datasource:
     driver-class-name: com.mysql.cj.jdbc.Driver
     username: root
-    password: 666666
+    password: 1234
     url: jdbc:mysql:///internship_platform
   redis:
     port: 6379
